test/extensions/filters/network/mongo_proxy/config_test.cc

```diff
@@ -95,8 +95,7 @@ TEST(MongoFilterConfigTest, InvalidExtraProperty) {
 }
 
 TEST(MongoFilterConfigTest, EmptyConfig) {
-  handleInvalidConfiguration(
-      "{}", R"(StatPrefix: \["value length must be at least " '\\x01' " runes"\])");
+  handleInvalidConfiguration("{}", R"(StatPrefix: value length must be at least 1 characters)");
```

Comment:
Nit: no need for a raw string literal. Same below.

Reply:
Done.

Reply:
I suspect the parens are going to make CI fail

Reply:
Yup.
I reverted the change, and then just removed the R and parens (don't know why I didn't do that the first place).
Either way, now done.